raven/src/main/java/com/getsentry/raven/connection/AbstractConnection.java

```diff
@@ -78,6 +86,9 @@ public final void send(Event event) {
             doSend(event);
             waitingTime = baseWaitingTime;
         } catch (ConnectionException e) {
+            for (EventSendFailureCallback eventSendFailureCallback : eventSendFailureCallbacks) {
+                eventSendFailureCallback.onFailure(event, e);
```

Comment:
Should this itself be wrapped in a `try`/`catch`? I'm a bit conflicted here, but my gut feeling is that the failure callback shouldn't propagate *more* errors in the case of it's own failure.